### ISSUE TITLE
[BITV] 9.1.3.1a/6.1 - All label elements of form elements were mistakenly marked as headings. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36912]
Since they are not headings, the heading markup should be removed. Only "Profile picture", "Details", and "Profile visibility" should be implemented as "h3" headings without labels inside. 

- [x] Fix
- [x] Backport

![0daff04b35b3395981d62d5f87436606](https://user-images.githubusercontent.com/1315170/221927177-7b26bd22-5ce4-4bf0-af21-654425064f93.png)

![ea791921f8d91989b62585d40226bf0f](https://user-images.githubusercontent.com/1315170/221927191-a68f058a-75bf-4a7c-8bcb-9928db1afd31.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-be75bbae42-v6-n1

</details>

**Comment by JuliaKirschenheuter** [comment]
Related (same) issue is here: https://github.com/nextcloud/server/issues/35626

**Comment by JuliaKirschenheuter** [comment]
Hi @michaelnissenbaum, 

i've changed a structure as you suggested. One open point is: i can't replace "profile picture" because this is a label which belongs to an input field which controls a file upload. I've created additional `<h3>` with "Your profile information". Is it ok in your eyes? Thank you!

![Screenshot from 2023-03-14 14-10-26](https://user-images.githubusercontent.com/6078378/225011022-df52137d-b264-44cf-9265-e2e79476d7ce.png)

**Comment by michaelnissenbaum** [comment]
@JuliaKirschenheuter  Looks good to me.
